### ISSUE TITLE
Problem w/ Bin refinement

### ISSUE THREAD

**Issue by TJrogers86** [bxlab/metaWRAP#147]

**Comment by ursky** [comment]
This likely comes from a typo or mistake in the command you typed. Can you provide the full command you used?

**Comment by TJrogers86** [comment]
Thanks for the quick response! The command was typed as follows:
metawrap bin_refinement -o CR_BIN_REFINEMENT -t 10 -A CR_INITIAL_BINNING/metabat2_bins/ -B CR_INITIAL_BINNING/maxbin2_bins/ -C CR_INITIAL_BINNING/concoct_bins/ -c -x

**Comment by TJrogers86** [comment]
I left c and x blank as the the default settings are 70 and 5.

**Comment by ursky** [comment]
The `-c` and `-x` parameters need to be followed by integers - the resired completion and contamination. Consult the tutorial and read the documentation, if this is confusing. For example, a typical run would look something like this:

```
metawrap bin_refinement -o CR_BIN_REFINEMENT -t 10 -A CR_INITIAL_BINNING/metabat2_bins/ -B CR_INITIAL_BINNING/maxbin2_bins/ -C CR_INITIAL_BINNING/concoct_bins/ -c 50 -x 10
```

**Comment by TJrogers86** [comment]
Nope, not confusing. I understand. The part that threw me off is that in 'A guide to analyzing metagenomic data with metaWRAP' tutorial, it states "By default, the minimum completion if 70%, and maximum contamination is 5%" so I wrongly assumed that no integers would be needed. Thank you for clearing that up. Will I need to delete the bin directories (A, AB, ABC, BC, and C) that the refinement module created and start over, or will typing in the command with the correct c and x cause it to pick up were it left off? Thanks again for your help and quick response.

**Comment by ursky** [comment]
Yes, I would re-start to be safe. And in general, the default values or parameters refer to the values used if the option is not indicated at all. As in:
 ```
metawrap bin_refinement -o CR_BIN_REFINEMENT -t 10 -A CR_INITIAL_BINNING/metabat2_bins/ -B CR_INITIAL_BINNING/maxbin2_bins/ -C CR_INITIAL_BINNING/concoct_bins/
```

**Comment by TJrogers86** [comment]
Oh, ok. Thanks! New grad student and new to using command lines. Thanks again. rookie mistake
